src/cmd_line/subparsers/file.ts

```diff
@@ -12,6 +12,26 @@ export function parseEditFileCommandArgs(args: string): node.FileCommand {
     let name = scanner.nextWord();
 
     return new node.FileCommand({
-        name: name
+        name: name,
+        position: node.FilePosition.CurrentWindow
     });
-}
\ No newline at end of file
+}
+
+export function parseEditNewFileInNewWindowCommandArgs(args: string): node.FileCommand {
+    return new node.FileCommand({
+        position: node.FilePosition.NewWindow
+    });
+}
+
+export function parseEditFileInNewWindowCommandArgs(args: string): node.FileCommand {
+    let name = "";
+    if (args) {
+        var scanner = new Scanner(args);
```

Comment:
Already change it to `var`. That's just my personal preference of using `let` all the time as people always get confused with the scoping of JavaScript's `var`, but here it's totally the same so use `var` is perfect either.